bash/solutii/marius.spinu/Compare/compare.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+    if [[ $# -ne 2 ]]; then
+        echo "Insufficient parameters"
+        exit 0
```

Comment:
Here it should be `exit 1` because it is not ok to have insufficient parameters.